File: Program73.java
import java.lang.*;
import java.util.*;

class ArrayX
{
    private int Arr[];
    private int iNo;

    public ArrayX(int iValue)
    {
        Arr = new int[iValue];
    }
    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.println("Enter element: "+(iCnt+1));
            Arr[iCnt] = sobj.nextInt();
        }

        System.out.println("Enter element to search");
        iNo = sobj.nextInt();
    }
    public boolean Search()
    {
        int iCnt = 0;
        boolean bFlag = false;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] == iNo)
            { 
                bFlag = true;
                break;
            }
        }
        return bFlag;
    }
}

class Program73
{
    public static void main(String arg[])
    {
       Scanner sobj = new Scanner(System.in);
       int iLength = 0; boolean bRet;

       System.out.println("Enter the value of elements");
       iLength = sobj.nextInt();

       ArrayX aobj = new ArrayX(iLength);

       aobj.Accept();

       bRet = aobj.Search();
       
       if(bRet == true)
       {
           System.out.println("Element is there in Array");
       }
       else
       {
        System.out.println("Element is not there in Array");
       }

    }
}

File: Program73.cpp
#include<iostream>
using namespace std;

class ArrayX
{
    private:
       int *Arr;
       int iSize;
       int iNo;
    
    public:
       ArrayX(int iNo1)
       {
           this->iSize = iNo1;
           Arr = new int[iNo1]; 
       }
       void Accept()
       {
           int iCnt = 0;

           for(iCnt = 0; iCnt < iSize; iCnt++)
           {
               cout<<"Enter element: "<<iCnt+1<<endl;
               cin>>Arr[iCnt];
           }

           cout<<"Enter element to search"<<endl;
           cin>>iNo;
       }
       bool Search()
       {
           int iCnt = 0;
           bool bFlag = false;

           for(iCnt = 0; iCnt < iSize; iCnt++)
           {
               if(Arr[iCnt] == iNo)
               {
                   bFlag = true;
                   break;
               }
           }
           return bFlag;
       }

};

int main()
{
    int iLength = 0;
    bool bRet;

    cout<<"Enter the value of constructor"<<endl;
    cin>>iLength;

    ArrayX aobj(iLength);

    aobj.Accept();

    bRet = aobj.Search();
    
    if(bRet == true)
    {
        cout<<"Element is there in Array"<<endl;
    }
    else
    {
         cout<<"Element is not there in Array"<<endl;
    }

    

}
